### PR TITLE
Portfolio-Desktop version 

### PR DESCRIPTION
**Pull request by Zuwairanajma**
Activities in this branch:
- I created a new branch.
- Added media queries for a min-width of 768px.
- Styled according to Figma template 2 desktop version.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -833,24 +838,283 @@ textarea::placeholder {
...
+  body {
+    width: 100%;
+    height: 100%;
+  }
+
+  .headbar {
+    display: flex;
+    width: 100%;
+    padding: 0 154px;
+  }
```
Comment (stanleySimeon) [inline styles.css]: According to the Figma design kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)..

> ## Your Design
>
>Screenshot 2023-06-15 at 12 48 36 PM
>
> ## The Figma Design
>
>Screenshot 2023-06-15 at 12 50 25 PM

File: styles.css [hunk 2]
```diff
@@ -833,24 +838,283 @@ textarea::placeholder {
...
+    overflow: none;
+    background-image: url(./PImages/Illustration-header@2x.png);
+    background-color: #1c1a19;
+    display: flex;
+    background-position: right top;
+    background-size: 25rem;
+    background-repeat: no-repeat;
+    flex-direction: row-reverse;
+    height: 100%;
+  }
```
Comment (stanleySimeon) [inline styles.css:891]: According to the Figma design kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)..

> ## Your Design
>
>Screenshot 2023-06-15 at 12 52 14 PM
>
> ## The Figma Design
>
>Screenshot 2023-06-15 at 12 53 54 PM

File: styles.css [hunk 3]
```diff
@@ -833,24 +838,283 @@ textarea::placeholder {
...
+    line-height: 32px;
+    margin: 12px 0 12px 150px;
+    width: 45.3%;
+    color: #f4f5f7;
+  }
+
+  .recent-works {
+    display: flex;
+    margin-left: 110px;
+  }
```
Comment (stanleySimeon) [inline styles.css:934]: According to the Figma design kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## Your Dsign
>
>Screenshot 2023-06-15 at 12 56 27 PM
>Screenshot 2023-06-15 at 12 58 49 PM
>
> ## The Figma Design
>
>Screenshot 2023-06-15 at 12 55 09 PM
>Screenshot 2023-06-15 at 12 58 01 PM

File: styles.css [hunk 4]
```diff
@@ -833,24 +838,283 @@ textarea::placeholder {
...
+
+  .about-myself {
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-left: 142px;
+    gap: 0 10rem;
+    z-index: 1;
+    margin-top: 120px;
+    margin-bottom: 100px;
+  }
```
Comment (stanleySimeon) [inline styles.css]: According to the Figma design kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## Your Design
>
>Screenshot 2023-06-15 at 1 00 42 PM
>Screenshot 2023-06-15 at 1 01 46 PM
>
> ## The Figma Design
>
>Screenshot 2023-06-15 at 1 01 32 PM
>Screenshot 2023-06-15 at 1 01 08 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Zuwairanajma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good PR desription
- Meaningful commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your readme file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Zuwairanajma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**